CM-747: Select a Dropdown Option Using Selenium WebDriver in C#

semirteskeredzic opened the pull request.

karimkod commented on dotnet-testing/SelectDropdownOptionUsingSeleniumWebDriver/SelectDropdownOptionUsingSeleniumWebDriver/Properties/launchSettings.json:
```diff
@@ -0,0 +1,37 @@
+      },
+      "dotnetRunMessages": true
+    },
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
```
This can be removed (only keep "https")

karimkod commented on dotnet-testing/SelectDropdownOptionUsingSeleniumWebDriver/SelectDropdownOptionUsingSeleniumWebDriver/Properties/launchSettings.json:
```diff
@@ -0,0 +1,37 @@
+    }
+  },
+  "profiles": {
+    "http": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "applicationUrl": "http://localhost:5208",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "dotnetRunMessages": true
+    },
```
This can be removed (only keep "https")

karimkod commented on dotnet-testing/SelectDropdownOptionUsingSeleniumWebDriver/SelectDropdownOptionUsingSeleniumWebDriverTests/LiveTest.cs:
Comments are not needed.

karimkod commented:
Hello @semirteskeredzic,
I am not able to run the test locally, is there anything specific I should do before running the test, apart from running the app? 
```
OpenQA.Selenium.WebDriverException: unknown error: result.webdriverValue.value list is missing or empty in Runtime.callFunctionOn response
```

semirteskeredzic commented:
Hi @karimkod , the issue probably lies in the discrepencies of the drivers e.g Google Chrome installed on the machine is probably 119.*.*.* while ChromeDriver is 118.*.*.*

The same applies to Edge. You can also change VersionResolveStrategy of the browserVersion from Latest to MatchingBrowser.

For reference: [https://stackoverflow.com/questions/77428873/im-getting-this-error-unknown-error-result-webdrivervalue-value-list-is-missin](https://stackoverflow.com/questions/77428873/im-getting-this-error-unknown-error-result-webdrivervalue-value-list-is-missin)

semirteskeredzic commented:
@karimkod , I've retested on my M1 machine and can't replicate the issue.
Google Chrome version was 116. and chromewebdriver was 118. I updated the Chrome on my machine to 119 and still couldn't replicate the issue.
Then I updated all packages to latest versions and still all tests pass. I then moved to Win11 machine and again weren't able to replicate the issue. To be on the safe side I've updated the edgewebdriver and added the driver for windows machines.
Please pull the latest code, build, run and test and let me know if you continue to have issue.

karimkod commented:
Thanks @semirteskeredzic, it's seems like the issue is fixed.
